code/modules/reagents/reagent_containers/spray.dm

```diff
@@ -20,7 +20,7 @@
 	var/stream_amount = 10 //the amount of reagents transfered when in stream mode.
 	var/can_fill_from_container = TRUE
 	amount_per_transfer_from_this = 5
-	volume = 250
+	volume = 50

```

Comment:
HHHHHHHHHHHHHHHHHAHAHAHAHAHAAAAAAAa
LOOK AT THE TOP OF HIS HEAD
LOOK AT HIS LIPS

Reply:
why did you nerf all spraybottles? this is unnecessary, just change the space cleaner bottle.